unix/vncpasswd/vncpasswd.cxx

```diff
@@ -99,6 +108,39 @@ static int encrypt_pipe() {
   return 0;
 }
 
+static int check_passwd_pwquality(const char *password)
+{
+	int r;
+	void *auxerror;
+	pwquality_settings_t *pwq;
+	pwq = pwquality_default_settings();
+	if (!pwq)
+		return -EINVAL;
+	r = pwquality_read_config(pwq, NULL, &auxerror);
+	if (r) {
+		printf(_("Cannot check password quality: %s \n"),
+			pwquality_strerror(NULL, 0, r, auxerror));
+		pwquality_free_settings(pwq);
+		return -EINVAL;
+	}
+
+	pwquality_set_int_value(pwq, PWQ_SETTING_MIN_LENGTH, 6);
+	pwquality_set_int_value(pwq, PWQ_SETTING_MAX_SEQUENCE, 8);
+	pwquality_set_int_value(pwq, PWQ_SETTING_MAX_REPEAT, 1);
+	pwquality_set_int_value(pwq, PWQ_SETTING_MIN_CLASS, 3);
```

Comment:
This doesn't seem proper to override. Isn't this configured by the system administrator in `/etc/security/pwquality.conf`?

Reply:
The default rules are in pwquality.conf
If specified in the code, the rules will be rewritten.